### PR TITLE
upgrade eks cluster master version

### LINKED ISSUE #8175: Upgrade Endpoint for EKS clusters
**User Story**
Upgrade Endpoint for Control Plane EKS clusters

**Acceptance criteria**

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita <harshita.sharma6174@gmail.com>

**What this PR does / why we need it**:
upgrade eks cluster master version

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #8175 

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: pkg/handler/v2/external_cluster/eks.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+limitations under the License.
+*/
+
+package externalcluster
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/service/eks"
```
Comment (zreigz) [inline pkg/handler/v2/external_cluster/eks.go:23]: please fix import

Comment (zreigz) [reply pkg/handler/v2/external_cluster/eks.go:23]: rest looks good

**Review by kron4eg — APPROVED** [review]
/approve

**Pull request merged**